charts/kubeclarity/templates/secret.yaml

```diff
@@ -1,9 +1,25 @@
 {{- $secretName := index .Values "kubeclarity-postgresql" "auth" "existingSecret" -}}
+{{- $postgresPassword := .Values.global.databasePassword | b64enc -}}
+{{- $dbHost := print (include "kubeclarity.name" .) "-postgresql" | b64enc -}}
+{{- $dbPort := index .Values "kubeclarity-postgresql" "service" "ports" "postgresql" | toString | b64enc -}}
+{{- $dbUser := index .Values "kubeclarity-postgresql" "auth" "username" | b64enc -}}
+{{- $dbName := index .Values "kubeclarity-postgresql" "auth" "database" | b64enc -}}
+{{- $dbSSLMode := index .Values "kubeclarity-postgresql" "auth" "sslMode" | b64enc -}}
 {{- if index .Values "kubeclarity-postgresql-external" "enabled" }}
   {{- $secretName = index .Values "kubeclarity-postgresql-external" "auth" "existingSecret" -}}
+  {{- $dbHost = index .Values "kubeclarity-postgresql-external" "auth" "host" | b64enc -}}
+  {{- $dbPort = index .Values "kubeclarity-postgresql-external" "auth" "port" | toString | b64enc -}}
+  {{- $dbUser = index .Values "kubeclarity-postgresql-external" "auth" "username" | b64enc -}}
+  {{- $dbName = index .Values "kubeclarity-postgresql-external" "auth" "database" | b64enc -}}
+  {{- $dbSSLMode = index .Values "kubeclarity-postgresql-external" "auth" "sslMode" | b64enc -}}
 {{- end }}
 {{- $secretKey := index .Values "kubeclarity-postgresql-secret" "secretKey" -}}
-{{- $postgresPassword := .Values.global.databasePassword | b64enc -}}
+{{- $usernameKey := index .Values "kubeclarity-postgresql-secret" "usernameKey" -}}
+{{- $hostKey := index .Values "kubeclarity-postgresql-secret" "hostKey" -}}
+{{- $portKey := index .Values "kubeclarity-postgresql-secret" "portKey" -}}
+{{- $databaseKey := index .Values "kubeclarity-postgresql-secret" "databaseKey" -}}
+{{- $sslmodeKey := index .Values "kubeclarity-postgresql-secret" "sslmodeKey" -}}
```

Comment:
you are missing the defaults here
maybe even better to set the defaults in the values file and forget about the defaults in the templates

Reply:
I see that the defaults are already set in the values, so no need for defaults in `charts/kubeclarity/templates/deployment.yaml`